addons/zeus/stringtable.xml

```diff
@@ -1037,41 +1049,51 @@
         </Key>
         <Key ID="STR_ACE_Zeus_ModuleLoadIntoCargo_DisplayName">
             <English>Load into Cargo</English>
+            <German>In Frachtraum laden</German>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_DisplayName">
             <English>Toggle NVGs</English>
+            <German>Ändere Nachtsichtgeräte</German>
             <French>Basculer JVN</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_ToggleNvgTitle">
             <English>NVG equipment</English>
+            <German>Nachtsichtgeräte</German>
             <French>Equipment de vision nocturne</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_ToggleNvgTitleTooltip">
             <English>Add or remove NVGs from units</English>
+            <German>Hinzufügen/Entfernen von Nachtsichtgeräten</German>
             <French>Ajouter ou retirer JVN des unités</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_ToggleNvgSide">
             <English>Toggle NVG target</English>
+            <German>Ändere Nachtsichtgeräte Ziel</German>
```

Comment:
Vom Gefühl her passt diese Übersetzung nicht so gut.

@alganthe iirc you added the NVG Zeus toggle function. Could you explain the function. It's hard to find a proper translation.